kubespawner/multispawner.py

```diff
@@ -417,16 +415,16 @@ def _wait_for_namespace(self, timeout=30):
 
     @gen.coroutine
     def _define_namespaced_account_objects(self):
```

Comment:
Should these be moved to objects.py where all K8s actions are defined? My version of objects.py already had all the required functions defined following the existing pattern.